### PR TITLE
Update metadata.md

### PR DESCRIPTION
**Pull request by wheresaddie**
update to v2.0.0

### CODE REVIEW

**Comment by jimmychu0807** [comment]
@wheresaddie I just directly push to update the json value in the article, reflecting Substrate v2 changes. Hope you don't mind 😄

**Comment by jimmychu0807** [comment]
Also, JSON syntax doesn't support commenting. That's why github diff viewer showing the JSON as red on `// ...`. I don't have a good solution on how to resolve this, and just let it be for now.

**Review by danforbes — APPROVED** [review]

File: docs/knowledgebase/runtime/metadata.md [hunk 1]
```diff
@@ -110,52 +110,51 @@ The hex blob that is returned by the JSON-RPCs `state_getMetadata` method starts
...
 Here is a condensed version of decoded metadata:
 
 ```json
-[
-  1635018093,
-  {
+{
+  "magicNumber": 1635018093,
+  "metadata": {
     "V11": {
```
Comment (thiolliere) [inline docs/knowledgebase/runtime/metadata.md:134]: This has been forgotten, should be v12 cc @wheresaddie

Comment (wheresaddie) [reply docs/knowledgebase/runtime/metadata.md:134]: good catch ty

Comment (wheresaddie) [reply docs/knowledgebase/runtime/metadata.md:134]: dropped to #718 PR thanks

**Pull request merged**